src/Talis/Html/Form/Validator/Container.php

```diff
@@ -38,18 +38,7 @@ public function add_required(\Talis\Html\Form\Element\aElement $element,string $
         return $this;
     }
     
-    /***
```

Comment:
why removing add_to_headers but leaving `private $headers` and  
```
foreach($this->headers as $name => $action){
            $header->$name = $action;
        }
```
Remove all of it, if we do not use it.